scripts/run_benches_for_runtime.sh

```diff
@@ -11,13 +11,9 @@ echo "[+] Running all benchmarks for $runtime"
 
 # shellcheck disable=SC2086
 cargo +nightly run $standard_args benchmark \
-    --chain "${runtime}-dev" \
-    --execution=wasm \
-    --wasm-execution=compiled \
-    --pallet "*" \
-    --extrinsic "*" \
-    --repeat 0 | \
-  sed -r -e 's/Pallet: "([a-z_:]+)".*/\1/' | \
+  --chain "${runtime}-dev" --list |\
```

Comment:
You may want to keep the previous indentation to easily distinguish the `cargo` from the following piped calls.